src/app/redux/UserSaga.js

```diff
@@ -152,6 +168,23 @@ function* usernamePasswordLogin({
         return;
     }
 
+    // return if already logged in using steem keychain
+    if (login_with_keychain) {
+        console.log('Logged in using steem keychain');
+        yield put(
+            userActions.setUser({
+                username,
+                login_with_keychain: true,
+                vesting_shares: account.get('vesting_shares'),
```

Comment:
This block needs to go below the fetch for account, missed it when i was implementing.

Reply:
https://github.com/steemit/wallet/pull/64/commits/f11e4995cddc1f5e26b038edd96b3cd073553cbb